src/_macosx.m

```diff
@@ -697,6 +698,41 @@ static CGFloat _get_device_scale(CGContextRef cr)
     Py_RETURN_NONE;
 }
 
+static PyObject*
+FigureManager_set_icon(PyObject* null, PyObject* args, PyObject* kwds) {
+    PyObject* icon_path;
+    static char* kwlist[3] = { "icon_path", NULL };
+    if (!PyArg_ParseTupleAndKeywords(args, kwds, "O&", kwlist,
```

Comment:
We should change to just `PyArg_ParseTuple` here if we aren't going to use the keywords.